index.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Quiz Works</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+		
+		<article class="salescard">
+			<header class="scheader">
+				<h1>Spanky Cheese</h1>
+			</header>
+
+				<section class="section1">
+					<div id="pic">
+						<img class="displayed" src="https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcSe3GXnP1NAiFVYPkqhhTLvkf_uPfrbkbI0GwDsqtvA0W_tKldfsg" height="60" width="60">
```

Comment:
all `img` tags must have an `alt` attribute. Also, don't use inline styles for height an width, use css.